code/game/turfs/simulated.dm

```diff
@@ -44,14 +44,13 @@
 	..()
 	if (istype(A,/mob/living/carbon))
 		var/mob/living/carbon/M = A
-		if(M.lying)	return

```

Comment:
All lying checks are in handle_slip() (that is called by slip()). Removing this enables us to make mobs lying down on a roller bed fall off the bed.